### PR TITLE
 F10: Pomodoro Timer Widget #14 

### PR DESCRIPTION
**Pull request by EwhanHan**
Added basic ass pomodoro timer;
show close on click and show when clicked in Dock

image

### CODE REVIEW

File: src/components/Pomodoro/Pomodoro.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+  const resetTimerTime = () => {
+    setTimerMinutes(25);
+    setTimerSeconds(0);
+  };
+
+  // Fired everytime timer is on/off and seconds state changes
+  useEffect(() => {
+    resetBreakTime();
+    let interval = timerActive
```
Comment (jhwan7) [inline src/components/Pomodoro/Pomodoro.js:60]: const

File: src/components/Pomodoro/Pomodoro.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+          } else {
+            // not 00:00
+            setTimerSeconds(timerSeconds - 1);
+          }
+        }, 1000)
+      : null;
+  }, [timerActive, timerSeconds]);
+
+  useEffect(() => {
+    let interval = breakActive
```
Comment (jhwan7) [inline src/components/Pomodoro/Pomodoro.js:85]: const

**Review by jhwan7 — APPROVED** [review]

**Comment by jhwan7** [comment]
seeing a positioning bug where its a little too below and the scrollbar appears,  
image

very minor issue

**Comment by EwhanHan** [comment]
> seeing a positioning bug where its a little too below and the scrollbar appears,  
> 
> image
> 
> 
> 
> very minor issue

You think overflow:hidden would be the right call here?

**Pull request merged**